scripts/taxonomy/classification.py

```diff
@@ -74,12 +74,12 @@ def main(
             # get the material label prediction result of the record
             # load the model predictor
             smp = SESARMaterialPredictor(sesar_model)
-            label, prob = smp.predict_material_type(
+            result = smp.predict_material_type(
                 thing.resolved_content
             )
 
             print(
-                f"Predicted (probability, label) : {label}, {prob}"
+                f"Predicted (probability, label) : {result.value}, {result.confidence}"
```

Comment:
This is really nit, but for future reference you can override `__repr__` on the class itself like so:

```
 def __repr__(self):
        return f"{result,value}, {result.confidence}"
```

then the callers would just need

`f"Predicted (probability, label) : {result}"`